src/modules/contract/components/Contract/Contract.tsx

```diff
@@ -1,36 +1,75 @@
 import { getAddress, isAddress, getCreateAddress } from 'ethers';
 import { useEffect, useState } from 'react';
 import { ABIDescription, FunctionDescription } from '@remixproject/plugin-api';
-import { useWeb3, useFhevmjs, Parameter, useRemix, LOCALSTORAGE_GATEWAY, formatParameters } from '../../../utils';
-import { Button, Label, TextInput } from '../../../common-ui';
+import {
+  useWeb3,
+  useFhevmjs,
+  Parameter,
+  useRemix,
+  LOCALSTORAGE_GATEWAY,
+  formatParameters,
+  LOCALSTORAGE_ACL_ADDRESS,
+  LOCALSTORAGE_KMS_VERIFIER_ADDRESS,
+} from '../../../utils';
+import { Button, Label, TextInput, Select } from '../../../common-ui';
 import { Inputs } from '../Inputs';
 import { ContractInterface } from '../ContractInterface';
 
 import './Contract.css';
 import { ContractFactory } from 'ethers';
 
-export type ContractItem = { address: string; abi: ABIDescription[]; name: string };
+import ZamaSepoliaConfig from '../../../../../config/ZamaSepoliaConfig.json';
+
+export type ContractItem = {
+  address: string;
+  abi: ABIDescription[];
+  name: string;
+};
 
 export const Contract = ({}) => {
   const { account, provider } = useWeb3();
   const { remixClient, info, log, error } = useRemix();
   const [name, setName] = useState<string>();
   const [abi, setAbi] = useState<ABIDescription[]>();
   const [bytecode, setBytecode] = useState<string>();
-  const [constructor, setConstructor] = useState<FunctionDescription | undefined>();
+  const [constructor, setConstructor] = useState<
+    FunctionDescription | undefined
+  >();
+
```

Comment:
Hardcode options for now. Later we'll select the good set of options base on the current chainId
`const options = networks["11155111"];`